src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
+import { Route } from 'react-router-dom';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
 
 const App = () => (
   <div>
-    <Navigation />
+    <Navigation  />
+    <Route exact path='/' component={Home}></Route>
+    <Route path='/about' component={About}></Route>
+    <Route path='/contact' component={Contact}></Route>
```

Comment:
This all looks great. See my Requested Improvements section for my comments on the self-closing `Route/>` tag.